shared/src/business/entities/cases/ElectronicPetition.ts

```diff
@@ -75,6 +77,7 @@ export class ElectronicPetition extends JoiValidationEntity {
   }
 
   static VALIDATION_RULES = {
+    attachmentToPetitionFiles: joi.array().items(joi.object().optional()),
```

Comment:
Should the entire array be optional?

Reply:
Yes it should be. Currently working on updating the optional field but specifically validating size of the pdfs using joi.